app/views/application/head/_meta_tags.html.erb

```diff
@@ -4,5 +4,6 @@
   <%= meta_tag('apple-mobile-web-app-capable','yes') %>
   <%= meta_tag('apple-mobile-web-app-status-bar-style','default') %>
   <%= meta_tag('apple-touch-fullscreen','yes') %>
+  <%= meta_tag('viewport', 'width=device-width, initial-scale=1, viewport-fit=cover') %>
```

Comment:
There is another version of this over in `_head`. Notably that one has all the same params except it's missing the `cover` option. But also, the other one is `viewport_tag()` whereas this is `meta_tag()`. Which is right?

Reply:
this was an oversight